techniques/systemSettings/networking/firewall/1.0/metadata.xml

```diff
@@ -0,0 +1,61 @@
+<TECHNIQUE name="Firewall">
+    <DESCRIPTION>Configures a simple firewall using nftables</DESCRIPTION>
+
+    <MULTIINSTANCE>false</MULTIINSTANCE>
```

Comment:
you cannot have several instance of this technique?

Reply:
No, it replaces the system's firewall configuration